### ISSUE TITLE
Torrentday no longer working

### ISSUE THREAD

**Issue by Fungie** [SickRage/SickRage#1892]
Enable debug logging in SickRage settings, reproduce the error (be sure to disable after the bug is fixed)

Branch/Commit: master/89676eb74a3fc9f765f17db291c9134df2537e38
OS: Linux-3.18.10-armv7l-with-glibc2.4 (openelec, raspberry pi 2)
What you did: Manually download show as it didn't automatically
What happened: couldn't find the show on torrentday, checked the website and its up. They may have tweaked their website
What you expected: to download automatically or to successfully manually download it

BTW: I seen the post from earlier today about torrentday but I am not in the UK and can access torrentday.com and classic.torrentday.com so its a separate problem.
Logs:

    2016-05-30 19:25:35 INFO SEARCHQUEUE-MANUAL-277165 :: Unable to find a download for:     [Silicon Valley - S03E06 - Bachmanity Insanity]

**Comment by SickRage2** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [x] Post debug logs, either inline (for smaller logs) or using gist
- [ ] Please wrap inline logs inside \`\`\` and \`\`\` for readability

Please make sure you also read [how to create an issue](https://github.com/SickRage/sickrage-issues/blob/master/README.md) and followed all of the steps. 

The title should describe your issue. Having "SR not working" or "I get this bug" for 100 issues, isn't really helpful. We will close issues if there isn't enough information.

Sometimes the devs may seem like grunts and respond with short answers. This isn't (always) because the dev hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let us know, and this bot may get updated to automatically answer you.

Thanks!

**Comment by duramato** [comment]
their site is having issues

**Comment by Fungie** [comment]
But I am able to download directly from it.

**Comment by bradcornford** [comment]
Looks like they have implemented a redirect on there login page classic.torrentday/t to there normal login, which has a capture.

**Comment by BKSteve** [comment]
I have the same problem, TD no longer being searched. This is probably because they've just done a major update.
Did a manual search and this is what the log file shows.

2016-06-02 17:58:11 DEBUG    SEARCHQUEUE-MANUAL-272644 :: [TorrentDay] :: POST URL: https://classic.torrentday.com/V3/API/API.php [Status: 302]
2016-06-02 17:58:11 DEBUG    SEARCHQUEUE-MANUAL-272644 :: [TorrentDay] :: With post data: cata=yes&search=12+Monkeys+S02E07&{edited}
2016-06-02 17:58:12 DEBUG    SEARCHQUEUE-MANUAL-272644 :: [TorrentDay] :: GET URL: https://classic.torrentday.com/logout.php [Status: 302]
2016-06-02 17:58:12 DEBUG    SEARCHQUEUE-MANUAL-272644 :: [TorrentDay] :: GET URL: http://classic.torrentday.com/ [Status: 301]
2016-06-02 17:58:12 DEBUG    SEARCHQUEUE-MANUAL-272644 :: [TorrentDay] :: GET URL: https://classic.torrentday.com/ [Status: 200]
2016-06-02 17:58:12 DEBUG    SEARCHQUEUE-MANUAL-272644 :: [TorrentDay] :: Requested a json response but response was not json, check the url: https://classic.torrentday.com/V3/API/API.php
2016-06-02 17:58:12 DEBUG    SEARCHQUEUE-MANUAL-272644 :: [TorrentDay] :: No data returned from provider


When checking the 'https://classic.torrentday.com/V3/API/API.php' in the browser - total blank page.

**Comment by duramato** [comment]
They "shutdown"  their classic login page, and as SR uses that because the normal has a recapture it does not work
Util the classic login page comes back or some other solution like an api is available,  I don't see it working again

**Comment by BKSteve** [comment]
I saw on TD site somebody mentioned Jackett.
Does Jackett work? Anybody know how to set it up in SR?

**Comment by duramato** [comment]
Set jackett up as you would add a newznab provider,  if you don't have any newznab enable blackhole and it will be sent for the torrent client you have enabled

**Comment by BKSteve** [comment]
I'm having troubles with Jackett itself on my DSM. Set it up, no problem come back later and not work??
I also wonder, if Jackett can access TD then how can they implement and SR not?

**Comment by BKSteve** [comment]
Jackett still a no go on the DSM, too beta.

**Comment by BKSteve** [comment]
Temporary work around using Custom RSS as torrent provider for TD
Get your feed from the RSS in TD https://www.torrentday.com/rss.php , should be
https://www.torrentday.com/t.rss?download;X;u=123456;tp=1234567890abcdef1234567890abcdef
and put the whole string in the Configure Custom Torrent Providers > RSS URL line. Leave the Cookies field blank.
Note there are no spaces and the X is whatever number comes for the categories you choose.

**Comment by trendless** [comment]
> Util the classic login page comes back or some other solution like an api is available, I don't see it working again

Unfortunate.

**Comment by sund00bie** [comment]
Sadly i too am having issues with torrentday :( This makes me sad

**Comment by Fungie** [comment]
The custom rss above doesn't work for me.

**Comment by blem189** [comment]
You need to generate your own RSS-url from the TD site (select direct download). Also make sure to select the categories you want in your RSS-feed, otherwhise you will get an error that it is empty. Mind that backlog search won't work with RSS.

**Comment by Fungie** [comment]
Think I am just going to start using usenet instead.

**Comment by duramato** [comment]
@Fungie There are good public indexes if you don't want to go "waste" money (rarbg, torrentproject,..) ,  that is..  if you don't live in a nazy country and you need a VPN.

**Comment by Qw-in** [comment]
Jackett seems to be working for me.

**Comment by dirkr07** [comment]
Jackett sounds nice..
Sadly If i'm correct a custom provider does not have the option to be a backlog provider? atleast I don't see any option to enable it as a backlog provider.

 (I personally only use TD as a backlog one)

**Comment by blem189** [comment]
Custom provider over RSS won't support backlog. 

Jackett uses the TD API (to my knowledge..), and it supports backlog. All you have to do is set up Jackett, open port in firewall, establish jackett connection to TD and add Jackett as a torznab provider. You will need a separate blackhole folder to transfer torrents from jackett to SR (you can still use daemon from SR to torrent client).

I linked this in a thread on the TD forum

**Comment by duramato** [comment]
Jackett does not use any API, it simpley does trickery with the ReCaptcha

and a few linguistic corrections @blem189 , `add Jackett as a torznab provider. ` , as Newznab provider .
`You will need a separate blackhole folder to transfer torrents from jackett to SR` If you don't use any newznab client just enable blackhole and the torrents are sent for your designated torrent client

**Comment by blem189** [comment]
Thanks for clearing that up @duramato

**Comment by BKSteve** [comment]
Forum Moderator - HotSauce, suggested SR to pm one of the site ops over on the TD forum site.
Forum post subject title - 'TD no longer working with Sickrage - Issues & Problems'

Jackett didn't function well on my DS, it would work then wouldn't, just flaky. 
RSS works and grabs 99% of my stuff, a little manual collection for those that aren't isn't a problem.

**Comment by Firemanjoe** [comment]
I hope that there is a solution that is found. I'm not saying that I'm lazy, but SickRage and TD sure make things easy!

**Comment by anothergene** [comment]
So why does Couch Potato still work with TD then?  Obviously something can be done.

**Comment by VeNoMouS** [comment]
None of the apps will be working , this is due to recapthca being set site wide..

refer https://github.com/junalmeida/Sick-Beard/issues/293

**Comment by sund00bie** [comment]
If i need to message an Admin. What would be the best thing to say ?

**Comment by labrys** [comment]
@sund00bie the TD admin's are aware of the situation and will not be doing anything about it.

**Comment by anothergene** [comment]
Don't know what to tell you. Couch potato still works. June 11th. 

CouchPotato: Snatched "Snow White and the Huntsman 2012 1080p Bluray DTS-HD MA 7.1 x264-BluntSlayer": Snow White and the Huntsman (2012) in 1080p from TorrentDay

**Comment by ripper1967** [comment]
That is correct, it may have grabbed it right now .they might not have added  the google ReCaptcha to that page yet,but they are and will be adding it soon. so i am reading.

**Comment by Firemanjoe** [comment]
Wow, that's too bad! Are they simply looking at increasing their site traffic? Do we think that SR can create a workaround or the Recaptcha is the end of it?

**Comment by Fungie** [comment]
Its a pain alright. I just bought the new raspberry pi (wanted it anyway),
put osmc on it since its a 'full' OS and used jackett. Works fine for tv
shows, haven't tried it with coach potato yet but assume it works.

**Comment by ripper1967** [comment]
it is being worked on

**Comment by Fungie** [comment]
Jackett integration would solve the problem very easily.

**Comment by duramato** [comment]
Jackett is supported..

**Comment by Fungie** [comment]
I know, I mean a more obvious way of enabling it for the casual user. I had
to set it up as a NZBindexer; unless I am on an older version.

**Comment by BKSteve** [comment]
@Fungie go take a look at the TD forum:
"TD no longer working with Sickrage - Issues & Problems"

nasco4 mid page 2 gives the following:
> To set up jackett in SR do the following
> 
> 1) Enable NZB under search settings
> 2) go to search providers
> 3) under the configure custom NZB tab add Jackett there.
> 4) Select a category! (If you don't do this it wont work.
> 5) Select update categories and if this does not populate anything save changes go to another settings window and back until it does. (This is a documented glitch)
> 6) When it does populate select 5000 (TV) once this is done you should be able to search without issue.
> 
> If any one is on Freenas I will post a quick setup guide to getting Jackett working in a freebsd jail under mono ASAP.

**Comment by bitzorro** [comment]
Any news on this, is there a possible fix comming soon?

**Comment by poopsmeow** [comment]
They will not be removing recapcha so this provider will need to be rewritten. @VeNoMouS has pretty much already done it for sickbeard even tho it seems to require some weird PHPSESSID stuff. Torrentday has well and truely fucked their site with always requiring capcha.

**Comment by VeNoMouS** [comment]
I have done it, it was commited the other day :)

**Comment by sund00bie** [comment]
@VeNoMouS Thanks!!

**Comment by stridervj** [comment]
@ poopsmeow: what version (fork url and branche) of Sickbeard are you using which have the Torrentday PHPSESSID stuff?

**Comment by VeNoMouS** [comment]
@stridervj mine.. https://github.com/VeNoMouS/Sick-Beard/

**Comment by stridervj** [comment]
@VeNoMouS : so I am trying to setup your version on my Synology. I need the following information:
Fork URL and Fork Branch. Can you help me with that?

**Comment by duramato** [comment]
Fork url I think is https://github.com/VeNoMouS/Sick-Beard.git and branch TorrentProviders but @VeNoMouS might be able to say with more certainty :P

**Comment by VeNoMouS** [comment]
The fork URL https://github.com/VeNoMouS/Sick-Beard.git , and there is only one branch, torrentProviders... 

I'm unsure of / if @echel0n , @labrys or any of the others will adapt my code to sickrage, I haven't heard from them in about a week, so as far as where a solution for sickrage at this stage, I do not know, so I can't comment.

**Comment by stridervj** [comment]
Thanks for the info. I will try it tonight and report back later.

**Comment by VeNoMouS** [comment]
To be completely honest, the workaround i put in place in sickbeard is crude at best, but I have no other solution to put forward at this time.

**Comment by labrys** [comment]
@VeNoMouS we will likely incorporate a version of your code in Medusa as a beta feature although the issues with the work-around required definitely aren't ideal, it is what it is until something better presents itself.  I appreciate your hard work on it.  I know people who rely on TorrentDay as a provider will be glad to have this available to them.

**Comment by VeNoMouS** [comment]
Hey man, That's awesome news for your users :) , yea I think we were all shafted by torrentdays change and lack of caring on their part of the impact it caused, I'm more than sure we weren't the only ones affected by said change.

**Comment by bradcornford** [comment]
@VeNoMouS Another solution, albeit just as crude, however not requiring ssh tunneling and binding, is to allow already set user session cookies to be attached to subsequent requests. From my minimal testing, it appears that only cookies "pass", and "uid" are required to authenticate. They also have a nearly infinite expiry of.

**Comment by p0psicles** [comment]
@bradcornford, you would have to use some

**Comment by abdiz1** [comment]
Hi

# Is there a final working solution to this problem? or do we all dump TD and find us a new provider?
why did TD add the capcha thingy... thats so lame

**Comment by p0psicles** [comment]
I think ditching it is the fastest way of removing there captcha.

**Comment by neoatomic** [comment]
Dont forget the real issue is TD here, they clearly want users to only access there site with an web-browser. And therefor adding the captcha. (Maybe advertisement revenues.?)

Regardless of that, personally i'm not in favor of complex fixes/workarounds. Its just not a good practice. And it doesn't really fix the issue at hand. At best its a temp workaround that TD will likely close on a later point. And then we are discussing the same problem again... :-1:

**Comment by VeNoMouS** [comment]
@bradcornford that is incorrect, as you require the PHPSESSID without it takes you back to the login screen

**Comment by VeNoMouS** [comment]
@bradcornford just checked again, and confirmed your theory, PHPSESSID is not required... at the end of the day, you could remove the PHPSESSID cookie set, but you would need to tunnel for the captcha ... AND/OR you could have additional to allow for manual cookie set, I think I may incorporate that into my branch.

**Comment by sund00bie** [comment]
Sorry for the rookie question BUT do we have a working solution for
sickrage?

**Comment by bitzorro** [comment]
Guess we are loosing a hole bunch of users to SickGear 😓
They are doing some heavy fishing inn the forums at TD..

**Comment by sund00bie** [comment]
If td works on sickgear I'm there

**Comment by abdiz1** [comment]
I do not believe TD works on sicggear due to the capcha

**Comment by p0psicles** [comment]
What sickgear does, is implemented a field, where you can enter the cookies (which you need to copy from your browser) manually. So you do the captcha in your browser. Make a note of the cookies that are required. And fill them in, in SG in the provider config. But because there is no cookie expiration, that works pretty well.

**Comment by labrys** [comment]
Until ofc they change something else or expire the cookies on the servers end.

**Comment by Voooo** [comment]
Since its open source, SickGear's solution would be implemented with a simple copy paste, IF it was viable. Since it isn't, i guess the solution isn't viable, or else the longest thread on this hub whould have been muted long time ago. But I'm just a beer drinking user and not a dev.

**Comment by p0psicles** [comment]
That solution is fine.  You just need a dev to spend about 2 hours on it.  Took me 2 doing it for Medusa.

**Comment by bitzorro** [comment]
Can the devs please just Tell uss if you are going to fix this? So we know if we have to maybye jump to Another train...
Been using TD since the start and not giving it up, if there is working options.

**Comment by echel0n** [comment]
Yes I fully intend to just been busy working

**Comment by Imnotgoodathis** [comment]
Well you can just use RSS while waiting. I just added TD as RSS when I noticed the issue, though two weeks of backlog to fix but thats no hurry for me atleast.

**Comment by echel0n** [comment]
How did you add as rss, I don't have those links

**Comment by Voooo** [comment]
you get the link from http://www.torrentday.me/rss.php once you chose your categories. However RSS only work for new releases.

**Comment by echel0n** [comment]
Requires login which I don't got

**Comment by Voooo** [comment]
We were under the impression that you were the one who was gonna fix this? I'm starting to ponder if you are qualified..

**Comment by p0psicles** [comment]
No need to ponder. I can guarantee you that he is not going to fix it for this repo. You are going to have to wait for @miigotu.

**Comment by echel0n** [comment]
You do understand I work and have a family and do this for free right ? Lol

**Comment by Voooo** [comment]
There are sickrage member tags and there isnt. Using RSS will not fix this. If you bothered to read the entired thread. You must work in the remotest part of the desert in Africa if you chip in after 1 month, "busy working, cannot type or read anything". Go away troll.

**Comment by echel0n** [comment]
Lol sure that'll so make we want to make this a priority ...

**Comment by Voooo** [comment]
Well, I have derailed this thread to much allready by debating. Everyone is still waiting for a solution, and I wont continue this pseudo debate.

**Comment by echel0n** [comment]
If the RSS links don't require login with captcha then yes this would work
but not for backlogs which leaves the alt solution being to grab the cookie
created by clicking on the captcha image and log into the site however that
highly relies on the cookie being nonexpiring. So don't go assuming I
didn't read the thread as I did when it first started but attitudes like
yours and subtle be digs suggesting I'm not capable of fixing this doesn't
exactly make me want to jump up and fix this or make it a priority as I
said I don't get paid got this and I have other priority's that I'm sure
are more worth my attention

**Comment by Qw-in** [comment]
Wouldn't this would just be a reimplimentation of something jackett already provides? The only issue with jackett (or the sickrage torznab implementation) is the lack of a number of settings such as minimum seeders, ratio, etc. 

I'm not sure if this is viable but a dedicated torznab option under custom torrents that implemented these setting would be perfect.

**Comment by echel0n** [comment]
Honestly my approach to this would be to add in jackett support instead of
trying to hack something together as I feel the overall results would be
much more stable and reliable in the long run.

**Comment by p0psicles** [comment]
Lol jacket support is already in SickRage

**Comment by echel0n** [comment]
Well then there yeah go

**Comment by VeNoMouS** [comment]
LOL , this thread is total *face palm*

**Comment by Djripper66** [comment]
I have yet to get any nzb working i have paided for like 10 different ones including recommended ones fortunately i was able to get refunds for most of them but i want to completely get rid of torrents and switch to nzbs but cant until i get it working

Sent from TypeApp<http://www.typeapp.com/r>


On 3 Jul 2016, at 10:39 am, VeNoMouS <notifications@github.com<mailto:notifications@github.com>> wrote:

LOL , this thread is total face palm

**Comment by Djripper66** [comment]
I keep getting xml errors for every paid or added custom nzb

Sent from TypeApp<http://www.typeapp.com/r>


On 3 Jul 2016, at 10:39 am, VeNoMouS <notifications@github.com<mailto:notifications@github.com>> wrote:

LOL , this thread is total face palm

**Comment by duramato** [comment]
@Djripper66 They need to be newznab based and correctly configured on SR.  And please don't use this issue for other things than TorrentDay not working, having this thread filled with spam does more harm than good.

**Comment by Djripper66** [comment]
I have tried on other threads but never got replys

Sent from TypeApp<http://www.typeapp.com/r>


On 3 Jul 2016, at 10:54 am, supergonkas <notifications@github.com<mailto:notifications@github.com>> wrote:

@Djripper66<https://github.com/Djripper66> They need to be newznab based and correctly configured on SR. And please don't use this issue for other things than TorrentDay not working, having this thread filled with spam does more harm than good.

**Comment by bitzorro** [comment]
I must say that im truly ashamed to see how we treat former companiens/devs...
I dont care what fueds there have been, that doesnt consern me at all! It just gave us users more options what to use..
And without @echelOn there vouldnt be any Sickrage in form we know! 
So just keep up the Good work all of you and then let the users choose the one that matches our needs the most,instead of thrashtalking!!

Regards bitzorro.

**Comment by p0psicles** [comment]
The thing is.. that echel0n has by no means any intent to PR against this branch or helping out the users of this branch (SickRage). And I have to honest, if I put some time into it, I could have PR'd a solution. But at least i'm not making any promises towards sr users, I cannot meet. Instead he's trolling in this thread. For which we've helped allot of users coming from that fork, who can finally experience a stable functioning SickRage, without occasional broken builds.

**Comment by bitzorro** [comment]
@pOpsicles If we are going to be picky, i rember there was som good trolling in the Sickrage forums and other places aswell when you guys left the old branch 😉

**Comment by bitzorro** [comment]
But stable it is! Il give you that 👍

**Comment by echel0n** [comment]
I was added into this thread from the start I never trolled onto it so
maybe get your facts straight.

Where did I promise anything and what would you have exactly to help your
members with if it wasn't for me coding it in the first place.

Show some respect and treat others how you wish to be treated, no wonder
why so many free projects die off these days!

**Comment by bitzorro** [comment]
@p0psicles i have to refrase ny comment about being stable..
On my part not so stable the last 30 days!!
Since TD is the only closed site ive used the last 5 years, this issue have made me and im sure other users to use other "open"providers resulting in unfinished downloads and password protected rars!
Thats not stable by any meens ! On my part.

**Comment by p0psicles** [comment]
Ok, this thread is going nowhere. To summarize the issue and possible solutions:
Issue: TD introduced captcha authentication. Although they use an almost never expiring cookie, that's required to bypass the captcha authentication.

Solution alternatives:
1. Proxy the google reCaptcha request through sickrage. Status: Has limitations, as it requires you to connect locally to your sickrage. And IMHO it's not a stable solution, and will raise more questions, then answered.
2. Add the cookie manually after doing the reCaptcha on your local browser, and copy/pase the uid and pass cookies into SR. status: I have a PR open with medusa. Will see if I can create a PR for SR any day now. I did some rework on medusa, because the cookie field already existed for rss torrent providers. And I wanted to not add multiple cookie fields. So I don't know if I can PR that one directly to SR.
3. Use jacket to add TD as an RSS provider. Limitation here is that your dependent on RSS. Jacket does some caching of results for you, but also that has limitations. Torznab support is already in SR.

**Comment by duramato** [comment]
A [fix](https://github.com/SickGear/SickGear/issues/718#issuecomment-230425241) has been issued and is now on the develop branch beeing tested. Now (when it goes with next master release)  you will have to add the cookies info to the provider's config,  as well as the login details like you used too.

**Comment by bitzorro** [comment]
@p0psicles under cookies at Td i find 3 different cookies:_cfduid, pass and uid !
whats the one i use SickRage??

**Comment by p0psicles** [comment]
You only need "pass" and "uid". So fill in something like: "uid=1342345;pass=asfdfasdfefasdfwfsfsf"

**Comment by ecoolaid** [comment]
**@p0psicles**  Thanks!  
Confirmed working (for me) on the develop branch including backlog search.

**Comment by Roughnick** [comment]
Thank you sooo much

**Comment by abdiz1** [comment]
Hi.  Can someone create a "how to set this up"  list?  
For us noobs.  Thanks

**Comment by Voooo** [comment]
"Funny isn't it? Funny isn't it?" To quote a great man.
Nothing happens for 35 days.
Some serious trolling starts, by it's purest form - provoking other people for the wanted response.
Then it's fix't instantly. (Most likley to shut the trolls up)
TY whoever fixed it.

**Comment by Imnotgoodathis** [comment]
Thanks :+1: 

> >  Voooo commented 3 hours ago
> 
> "Funny isn't it? Funny isn't it?" To quote a great man.
> Nothing happens for 35 days.
> Some serious trolling starts, by it's purest form - provoking other people for the wanted response.
> Then it's fix't instantly. (Most likley to shut the trolls up)
> TY whoever fixed it.

Since this solution was posted almost instantly on few places I guess they were looking at other options first.

**Comment by labrys** [comment]
@Voooo also the solution, and several alternatives, were also tested first on other forks such as @VeNoMouS [SickBeard](https://github.com/VeNoMouS/Sick-Beard) fork, the [SickGear](https://github.com/SickGear/SickGear) fork, and @p0psicles fix on the [Medusa](https://github.com/pymedusa/sickrage) fork.  Then @p0psicles was kind enough to adapt his work and PR it here as well.  So instead of complaining you should thank him for his hard work (and all of the other devs who contributed to a solution).

**Comment by BKSteve** [comment]
I switched to develop branch to try this out, don't see any changes on the 
Search Providers>Provider Options tab, for TorrentDay, are there any?

With the old master
Configure Custom Torrent Providers > RSS URL line. 
https://www.torrentday.com/t.rss?download;X;u=123456;tp=1234567890abcdef1234567890abcdef
already worked, no backlog is all.

Now when trying to switch back to Master branch I get:
Can't switch branch as this will result in a database downgrade.

**Comment by sund00bie** [comment]
i confirm TD does NOT work on sickgear

**Comment by Hellowlol** [comment]
Not derail the thread completely but torrentday is absolute shit so i wonder what the fuzz is about :D

**Comment by p0psicles** [comment]
Sorry guys, I made a typo. I'm preparing a fix for it. Will PR that today.
You will notice it if:
1. You've checked out develop
2. You've filled in the cookie string
3. After configuring it you have rebooted.

Result: It will show the cookie field in the Api Key field.

**Comment by labrys** [comment]
@sund00bie I was under the impression it had been I've corrected my statement.

**Comment by labrys** [comment]
@sund00bie SickGear does work, the fix is in the develop branch and has been for quite a while.

**Comment by sund00bie** [comment]
I followed the arch guide on sg. Which pulls from master.

What command do I need to run to pull from dev?

**Comment by labrys** [comment]
@sund00bie If you're running SG you should ask on the SG issue tracker.

**Comment by VeNoMouS** [comment]
*unsubscribes from this spam*

**Comment by miigotu** [comment]
Please keep the discussion here to SickRage only. Hub-bombing for other projects just creates more spam. If you are using another project/fork or are promoting them do it on their issue tracker, thanks.

**Comment by bitzorro** [comment]
Are any others having issues not finding any torrents?
Lots of my weekly searches that used to work before the TD Change is not being found!

**Comment by qldhighway** [comment]
Any idea when this fix will be committed to the master branch?

**Comment by miigotu** [comment]
As soon as it is incubated in develop for a few days, one typo was already found and fixed but I have not heard feedback as to any other issues yet.

**Comment by ecoolaid** [comment]
FYI - when I switched back to Master from Develop, the new cookie field was blank.  Not a big deal obviously, if that is just an anomaly of when I switched branches or something.

**Comment by Koeniee** [comment]
Hi everyone,

Can somebody explain how to get this working? I have filled in my TD username and password in the Cookie field like this: uid=[username];pass=[password], where [username] is of course replaced by my real username (and [password] with my real password).

But the warning log still says:  [TorrentDay] :: Failed to login, check your cookies.

These are my steps:
1. Go to Sickrage
2. Go to Search Providers
3. Go to Provider Priorities
4. Check Torrentday
5. Go to Provider Options
6. Enter my TD user name
7. Enter my TD password
8. Enter Cookie information as described above
9. Click Save Changes
10. Restart SR

What am I doing wrong? Help would be really appreciated :)

**Comment by duramato** [comment]
You need to get your TD cookies,  not your login info.

**Comment by Koeniee** [comment]
Thank you for the quick reply, but where do I get my cookie information?

**Comment by sund00bie** [comment]
https://github.com/SickGear/SickGear/wiki/Frequently-Answered-Questions

It's on there

**Comment by Koeniee** [comment]
Awesome support, it is working again! Thank you both for pointing me in the right direction, this really makes me happy :).

**Comment by sund00bie** [comment]
It must be said though.  Thats a sickgear page (not sickrage)

On 25 Jul. 2016 07:46, "Koeniee" <notifications@github.com> wrote:

Awesome support, it is working again! Thank you both for pointing me in the
right direction, this really makes me happy :).

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/SickRage/SickRage/issues/1892#issuecomment-234804473>,
or mute the thread

**Comment by miigotu** [comment]
@sund00bie it works in SickRage, THIS SickRage.

**Comment by sund00bie** [comment]
@miigotu yes.  I am aware.  Just pointing out that the instructions for
using the cookie is hosted on the sickgear page.

That is all.

**Comment by miigotu** [comment]
Yeah I missed the previous comment somehow and then deleted my comment =P, ignore me just this once haha

**Comment by abdiz1** [comment]
Sickrage and TD is working.  But what about download station?  It is the DS that logg into TD and grabs the torrent files.  The DS only  have username and password.  And when testing the connection,  its refused by TD due to the Capcha. 

How can i get pass this?  Is there a fix to this?  Or maybe a bypass?  Can someone please  guide me  so i can get it to work again.. ☺ 
When i have a complete TD work arround,  i will make a guide so other users can use.

**Comment by dirkr07** [comment]
@abdiz1  what do you mean?
Download Station (DS) never logs in to TD. SickRage queries TD to get the .torrent file and either sends it directly to DS ( you authenticate to your DS from SickRage with a user that exists in DS) or you use the blackhole method.

DS never goes directly to TD to grab a torrent file through SickRage.

If you use the RSS feed of TD directly in DS than that has nothing to do with any SickRage functionality.

**Comment by abdiz1** [comment]
1: So i just ignore the # invalid username or password message that comes from DS?
2: if i do a manualy torrent search,  i only get a quarary from KAT and never from TD. (i have setup two sources: TD and KAT in that order) 

Any clues why or what is wrong?

**Comment by duramato** [comment]
Are you using this SR? And I don't know how you get results from kat,  when it is gone. 
And afaik SR is the one that does the torrent downloading.

**Comment by abdiz1** [comment]
Yes. Am using SR. Damn... KAT is down... Thats why my entire system is silent.  Lol. 
ill search for alternativ sites. But i use TD and IPTorrents. I think my settings are all messed up.... System cleanup here i come. Thanks for all support

**Comment by KaMoS69** [comment]
I can confirm that TD is working with SR again. I was struggling to get this right, kept getting errors. I found that the problem was a space in the cookies line. I removed all the spaces and it worked.

I used this guide to find my cookies: https://github.com/zone117x/Jackett/wiki/Finding-cookies

**Comment by BKSteve** [comment]
The cookie thing and finding it by going here and there, watch this video, etc is painful.

Go to TD https://www.torrentday.com/rss.php, and get an RSS on the screen should be
https://www.torrentday.com/t.rss?download;X;u=123456;tp=1234567890abcdef1234567890abcdef

Then grab the last bit
u=123456;tp=1234567890abcdef1234567890abcdef

Change u to uid and tp to pass
uid=123456;pass=1234567890abcdef1234567890abcdef
Put this in SR.

Perhaps @miigotu can implement so you can just put the string from the RSS - u=123456;tp=1234567890abcdef1234567890abcdef

**Comment by BKSteve** [comment]
All the sites to get cookies with fancy pictures and videos.

Firefox - on TD site.
F12, then Network Tab, HTML Tab (Reload button)
Select The page you're on Left side
Look in right side for Cookie, even use the search.
Just need the uid and pass, none of the rest.

**Comment by miigotu** [comment]
There are a million results just googling about how to get cookies from any browser. This discussion is closed.
